### PR TITLE
Imagine TODO API using Laravel PHP

### PR DESCRIPTION
**Pull request by fuadit**

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -18,4 +18,5 @@ If you require any assistance or support, feel free to let us know. Join our Sla
 Ohha! The participation list is totally empty. Hurry up if your want to have your name
 here before anyone else.
 
-- Example project by @GITHUB_USER_HANDLE. Link.
+- imagine_todo_api by @fuad-works.  [Link](https://github.com/fuad-works/imagine_todo_api).
```
Comment (bassem-i) [inline backend/README.md]:
```suggestion
- Imagine Todo API by @fuad-works. [Link](https://github.com/fuad-works/imagine_todo_api).
```

File: backend/README.md [hunk 2]
```diff
@@ -18,4 +18,5 @@ If you require any assistance or support, feel free to let us know. Join our Sla
 Ohha! The participation list is totally empty. Hurry up if your want to have your name
```
Comment (bassem-i) [inline backend/README.md]:
```suggestion

```

**Review by bassem-i — COMMENTED** [review]
Thanks @fuad-works for contributing! 💛 We hope that your work will be seen by Imagine fellows all around the world & motivate them to do the same! How was the task for you? Feel free to share your repo on Imagine's Slack community and ask other fellows for their feedback.

File: backend/README.md (hunk 2 above)
Comment (fuadit) [reply backend/README.md]: You are right, I will do that right away

File: backend/README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ If you require any assistance or support, feel free to let us know. Join our Sla
 
 ## Participation list
 
-Ohha! The participation list is totally empty. Hurry up if your want to have your name
 here before anyone else.
```
Comment (bassem-i) [inline backend/README.md]: Oh, your missed a bit here. 
```suggestion

```

**Pull request merged**